src/cmd/troubleshoot/logger.go

```diff
@@ -75,20 +84,30 @@ func errorWithMessagef(err error, format string, v ...interface{}) error {
 func (dtl troubleshootLogger) Init(_ logr.RuntimeInfo) {}
 
 func (dtl troubleshootLogger) Info(level int, message string, keysAndValues ...interface{}) {
+	var msg string
+
 	switch level {
 	case levelNewTest:
-		dtl.logger.Info(prefixNewTest+message, keysAndValues...)
+		msg = prefixNewTest + message
```

Comment:
```suggestion
		message = prefixNewTest + message
```
message can be changed directly without the use of a variable

Reply:
Although it doesn't have any side-effects here, I'm not a friend of re-using parameters as local variables. It changes the semantic from "input" to "variable" and if later on some decides to change the signature to using a pointer (admittedly not very likely here), it can lead to odd and hard to spot side-effects.

Reply:
If a string is passed to a function as a pointer with the intention of propagating this mutation that would raise at least one red flag in my mind.

Reply:
In general I agree with Stefan, would not re-use parameters. 

For the current lines it does not matter, as the msg will be returned directly in my new changes.